extensions/binding/org.eclipse.smarthome.binding.wemo/src/main/java/org/eclipse/smarthome/binding/wemo/handler/WemoCoffeeHandler.java

```diff
@@ -106,6 +106,7 @@ public void run() {
 
             } catch (Exception e) {
                 logger.debug("Exception during poll : {}", e);
+                updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR);
```

Comment:
For communication errors, it is always helpful to add `e.getMessage()` as a third parameter.

Reply:
Alright, done.